### PR TITLE
Ditch logentries gem, explicitly log to STDOUT in prod/stage

### LINKED ISSUE #18: Remove Logentries in favor of standard 12factor logging
We should remove the `le` gem in favor of logging to STDOUT.

### PR DESCRIPTION
**Pull request by smudge**
### Summary

I think the Rails 5 stuff broke some logging. This is an attempt to fix that, but we should confirm in stage that this is the correct solution.

Also this resolves #18 - we don't need the gem dependency.

/domain @Betterment/test_track_core 
/platform @jmileham @aburgel @samandmoore

### CODE REVIEW

File: config/environments/demo.rb [hunk 1]
```diff
@@ -71,5 +71,6 @@
   # Set to :debug to see everything in the log.
   config.log_level = :info
 
-  config.logger = ActiveSupport::TaggedLogging.new(Le.new(ENV['LOGENTRIES_TOKEN'])) if ENV['LOGENTRIES_TOKEN'].present?
+  # Use a different logger for distributed setups.
+  config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
```
Comment (samandmoore) [inline config/environments/demo.rb]: can we throw this in `application.rb` since it's the same errwhere?

**Comment by samandmoore** [comment]
<<domain tafn

**Comment by aburgel** [comment]
<< domainlgtm platformlgtm

File: config/environments/demo.rb (hunk 1 above)
Comment (smudge) [reply config/environments/demo.rb]: Well, it's not the same in `test` and `development`. But I could do the inverse -- is there a `Logger.new` for writing to `logs/X.log`?

Comment (aburgel) [reply config/environments/demo.rb]: except for `development` and `test`?

Comment (samandmoore) [reply config/environments/demo.rb]: oh, right. derp. nvm, let's roll with this.

**Comment by samandmoore** [comment]
domainlgtm

**Pull request merged**